config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-    jitsiMeetURL: "https://meet.jit.si/"
+    jitsiMeetURL: "https://meet.jit.si/",
+    handleAuthorization: false
```

Comment:
After some discussions recently it was decided that Jitsi Meet's default behavior will be to handle the authorization itself. This PR just follows Jitsi Meet's behavior and makes the jitsi-meet-electron project compatible with the latest versions of meet.